config/importmap.rb

```diff
@@ -0,0 +1,8 @@
+# Pin npm packages by running ./bin/importmap
+
+pin 'application', preload: true
+
+pin 'datatables'
+pin 'stupidtable'
+
+pin_all_from 'app/javascript/tylium', under: 'tylium'
```

Comment:
Unless I'm missing something, `app/javascript/tylium` doesn't exist but `app/assets/javascripts/tylium` does. Is this functioning as intended?